lib/HttpClient.js

```diff
@@ -46,11 +51,20 @@ if (!nativeKeepAlive) {
     httpsMaxSockets = Math.min(httpsMaxSockets, 1024);
 }
 
+// require('http').STATUS_CODES has a http 429 error that in < node 4 was known
+// as TooManyRedirects. This has been rectified as TooManyRequests.
+// in the case of restify-clients, we will retain this error type given that
+// users can specify maximum number of redirects to follow.
+if (!errors.TooManyRedirectsError) {
```

Comment:
This hasn't changed, I just moved it to the top of the file after the global requires and before function declarations. Also added a comment.